test/libsolidity/semanticTests/inlineAssembly/blobbasefee_shanghai_function.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+    function f() public view returns (uint ret) {
+        assembly {
+            let blobbasefee := sload(0)
```

Comment:
Should this be something like:
```suggestion
            sstore(0, 1)
            let blobbasefee := sload(0)
```
Or just:
```suggestion
            let blobbasefee := 1
```
Because the return of `sload(0)` will always be zero, since nothing was stored at slot 0. I mean, not that this is really very relevant for the test, but just for consistency with function `g()`.

Reply:
Sure, makes sense.